target/linux/ath79/image/mikrotik.mk

```diff
@@ -37,6 +37,20 @@ define Device/mikrotik_routerboard-922uags-5hpacd
 endef
 TARGET_DEVICES += mikrotik_routerboard-922uags-5hpacd
 
+define Device/mikrotik_routerboard-952ui-5ac2nd
+  $(Device/mikrotik_nor)
+  SOC := qca9533
+  DEVICE_MODEL := RouterBOARD 952Ui-5ac2nD (hAP ac lite)
+  DEVICE_PACKAGES += kmod-ath10k-ct-smallbuffers ath10k-firmware-qca9887-ct
+  IMAGE_SIZE := 3712k
+  SYSUPGRADE_IMAGE_SIZE := 16256k
+  IMAGE/sysupgrade.bin := append-kernel | kernel2minor -s 1024 -e | \
+	pad-to $$$$(BLOCKSIZE) | append-rootfs | pad-rootfs | \
+	check-size $$$$(SYSUPGRADE_IMAGE_SIZE) | append-metadata
+  SUPPORTED_DEVICES += rb-952ui-5ac2nd
+endef
+TARGET_DEVICES += mikrotik_routerboard-952ui-5ac2nd
```

Comment:
This whole patch is wrong. See how it's done for the other NOR devices. You shouldn't need to redefine IMAGE/sysupgrade.bin or define SYSUPGRADE_IMAGE_SIZE

Reply:
Can you show working patch? I want to compile latest OpenWrt for hAP AC Lite and test it.